db/migrate/20220702123618_create_metais.rb

```diff
@@ -0,0 +1,90 @@
+class CreateMetais < ActiveRecord::Migration[6.0]
+  def up
+    execute 'CREATE SCHEMA metais'
+
+    create_table 'metais.projekty' do |t|
+      t.string :uuid, null: false
+      t.string :nazov
+      t.text :popis
+      t.datetime :datum_zacatia
+      t.datetime :termin_ukoncenia
+      t.string :schvalovaci_proces
+      t.string :kod_metais
+      t.string :typ_investicie
+      t.string :zdroj
+      t.string :financna_skupina
+      t.string :prijimatel
+      t.string :faza_projektu
+      t.decimal :suma_vydavkov
+      t.decimal :rocne_naklady
+      t.string :ref_id
+      t.string :program
+      t.string :status
```

Comment:
Vsetko moze byt null?

Reply:
nakoniec nie všetko, ale všetko okrem názov, kódmetais a typ investicie. Sťahoval som celé matais a tak to bolo.